src/Item/Item.model.ts

```diff
@@ -70,25 +70,38 @@ export class Item extends Model<ItemAttributes> {
 
   // PAGINATED QUERIES
 
-  static findAllItemsByAddress(
-    thirdPartyIds: string[],
+  static findItemsByAddress(
     address: string,
-    limit: number = DEFAULT_LIMIT,
-    offset: number = 0
+    thirdPartyIds: string[],
+    parmas: {
+      collectionId?: string
+      limit?: number
+      offset?: number
+    }
   ) {
+    const { collectionId, limit, offset } = parmas
     return this.query<ItemWithTotalCount>(SQL`
       SELECT items.*, count(*) OVER() AS total_count
         FROM ${raw(this.tableName)} items
-        JOIN ${raw(
+        LEFT JOIN ${raw(
```

Comment:
Would you mind explaining why do we need a LEFT JOIN here? We want all of the returned values to match any of the clauses we're using.

Reply:
The thing is that the `JOIN` statement is joining the tables by `collection.id`.. Without the `LEFT` we wouldn't get the items that don't have a collection associated as the results will be just the ones with one. 

JOIN:
<img width="1498" alt="image" src="https://user-images.githubusercontent.com/8763687/163158138-3eb6e61f-a5bd-4001-95bc-9e1aa796f54a.png">

LEFT JOIN:
<img width="1533" alt="image" src="https://user-images.githubusercontent.com/8763687/163158077-0fe71f86-0dde-4655-8ef6-00bcfc8be3ec.png">